include/ddc/kernels/splines/spline_builder.hpp

```diff
@@ -270,20 +342,23 @@ class SplineBuilder
     }
 
     /**
-     * @brief Build a spline approximation of a function.
+     * @brief Compute a spline approximation of a function.
```

Comment:
Elsewhere you have removed "spline approximation". Once we agree on the best terminology it should be changed here too

Reply:
Let's keep it

Reply:
I restored it (but I'll always be surprised by the talent of mathematicians to use confusing words to speak about simple concepts :p)